### PR TITLE
OS support and templates for PR/Issues

### PR DESCRIPTION
**Pull request by chukka**
## OS support 
* Current ansible collection only supports ubuntu flavours
* Centos/RHEL and SELinux support is coming soon, stay tuned :)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - Describe *in detail* the feature/behavior/change you'd like to see.
+
+In both cases, be ready for followup questions, and please respond in a timely
+manner.  If we can't reproduce a bug or think a feature already exists, we
+might close your issue.  If we're wrong, PLEASE feel free to reopen it and
+explain why.
+-->
+
+**Which installer**:
+
```
Comment (eldada) [inline .github/ISSUE_TEMPLATE.md:26]: Add new line

**Review by eldada — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (chukka) [reply .github/ISSUE_TEMPLATE.md:27]: Done

**Comment by wdrury-uk** [comment]
Is there an update please as new version of the collection advised on my other issues was December 2020.  Whats the new ETA of the next collection version fixing this issue?

**Comment by chukka** [comment]
Please see comments in  #57

**Comment by wdrury-uk** [comment]
Thanks @chukka for the update.  We will plan our next internal platform release to coincide few week sprint.  Looking forward to the improvements.

**Pull request merged**